Xamarin.Essentials/Types/DeviceCalendar.shared.cs

```diff
@@ -22,11 +22,44 @@ public interface ICalendar
         bool IsReadOnly { get; set; }
     }
 
-    public class Event
+    public interface IEvent
     {
-        public int Id { get; set; }
+        string Id { get; set; }
+
+        string CalendarId { get; set; }
+
+        string Title { get; set; }
+
+        string Description { get; set; }
+
+        string Location { get; set; }
+
+        bool AllDay { get; set; }
+
+        long? Start { get; set; }
+
+        long? End { get; set; }
+
+        bool HasAlarm { get; set; }
+
+        bool HasAttendees { get; set; }
+
+        bool HasExtendedProperties { get; set; }
+
+        string Status { get; set; }
+
+        List<Reminder> Reminders { get; set; }
```

Comment:
For the moment I think we should remove Reminders and RecurranceDate and RecurranceFrequency - I'm not sure whether reminders map well across all platforms and we should be exposing nicer properties for recurring events

Reply:
Altered in next PR